api/v1alpha1/cat.go

```diff
@@ -4,11 +4,18 @@ import (
 	corev1 "k8s.io/api/core/v1"
 )
 
-type Cat struct{}
+type Cat struct {
+	AbstractStep `json:",inline" protobuf:"bytes,1,opt,name=abstractStep"`
+}
 
 func (m Cat) getContainer(req getContainerReq) corev1.Container {
+	resources := standardResources
```

Comment:
Do we need this, doesn’t the default populate this?

Reply:
no